test/GPv2Encoding.test.ts

```diff
@@ -75,7 +75,7 @@ describe("GPv2Encoding", () => {
       for (let i = 0; i < tradeCount; i++) {
         await encoder.signEncodeTrade(
           { ...sampleOrder, appData: i },
-          0,
+          {},
```

Comment:
Considering how it's used, maybe it makes sense to make it into an optional parameter to the end, similar to how ethers.io allows you to optionally specify `{gas:..., gasPrice:..., from:...}`?